sky/clouds/gcp.py

```diff
@@ -457,6 +459,22 @@ def get_credential_file_mounts(self) -> Dict[str, str]:
         credentials[GCP_CONFIG_SKY_BACKUP_PATH] = GCP_CONFIG_SKY_BACKUP_PATH
         return credentials
 
+    def get_cloud_user_identity(self) -> Optional[str]:
+        # Returns the email address of the currently activated user.
+        try:
+            account = _run_output('gcloud auth list --filter=status:ACTIVE '
+                                  '--format="value(account)"')
+        except subprocess.CalledProcessError as e:
+            with ux_utils.print_exception_no_traceback():
+                raise exceptions.CloudUserIdentityError(
+                    'Failed to get GCP user identity with unknown '
+                    f'exception: {type(e)} {e}') from e
+        if not account:
+            with ux_utils.print_exception_no_traceback():
+                raise exceptions.CloudUserIdentityError(
+                    'No GCP account is activated.')
+        return f'{account}@{self.get_project_id()}'
```

Comment:
It's confusing to have two `@`, especially if these strings can get displayed. How about `a@b.com [project_id={project_id}]`?